Retsinformation.js

```diff
@@ -0,0 +1,191 @@
+{
+	"translatorID": "feef66bf-4b52-498f-a586-8e9a99dc07a0",
+	"label": "Retsinformation",
+	"creator": "Roald Frøsig",
+	"target": "^https?://(?:www\\.)?retsinformation\\.dk/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2016-02-02 13:53:31"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2016 Roald Frøsig
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url){
+	if (ZU.xpathText(doc, '//div[@class="wrapper2"]/table[@id="ctl00_MainContent_ResultGrid1"]') && getSelectItems(doc, url).length) {
+		return "multiple";
+	} else if (url.indexOf("R0710") != -1) {
+		type = getType(doc, url);
+		return type;
+	} else return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		var items = new Object();
+		var titles = getSelectItems(doc, url);
+		for (i = 0; i < titles.length; i++) {
+			items[titles[i].href] = ZU.trimInternal(titles[i].text);
+		}
+		Z.selectItems(items, function(selectedItems) {
+			if (!selectedItems) return true;
+			var urls = new Array();
+			for (var i in selectedItems) {
+				urls.push(i);
+			}
+			ZU.processDocuments(urls, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function getSelectItems(doc, url) {
+	if (/(R0900|R0920|R0930)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[1]/a[1]');
+	} else if (/(R0210|R0310)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[2]/a[1]');
+	} else if (/R0700.aspx\\?res/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[3]/a[1]');
+	} else if (/(R0220|R0415|R0700)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[4]/a[1]');
+	}
+	return titles;
```

Comment:
What if none of the `if` statements above match?
just a suggestion, but you can replace all instances of `var titles = ` with `return `. Then return `[]` at the end if nothing matches.